universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -51,15 +54,16 @@
 
   @Constraints.Required private LifecycleStage lifecycleStage;
 
-  private @DbJsonB ImmutableList<QuestionOption> questionOptionsWithLocales;
+  // questionOptions is the legacy storage column for multi-option questions. A few questions
```

Comment:
nit: ah, formatting seems to have been awkward here. Mind making this a little more readable?

Reply:
done